lib/ipam/ipam_block_reader_writer_test.go

```diff
@@ -270,6 +275,7 @@ var _ = testutils.E2eDatastoreDescribe("IPAM affine block allocation tests", tes
 						defer GinkgoRecover()
 
 						testhost := "single-host"
+						applyNode(bc, testhost, nil)
```

Comment:
Same here - we're going to apply the node 4 times since we're inside a loop. Let's do this outside of the loop.

Reply:
addressed 9fe3116